hor-fun.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>The Best Chicken Rice Recipes!</title>
+  </head>
+  <body>
+    <h1>Hor Fun</h1>
+
+    <h3> Ingredients </h3>
+        <ul>
+            <li>400 g Thick Rice Noodles Hor Fun</li>
+            <li>>150 g Pork slices</li>
+            <li>12 pcs Prawns devein</li>
+            <li>5 cloves Garlic minced</li>
+            <li>5 stalks Bok Choy</li>
+            <li>1 qty Egg</li>
+            
+        </ul>
+
+    <h3>Preparation time </h3>
+    <p> It takes at most 1 hour to prepare this dish. </p>
+    <h3> Instructions </h3>
+        <ol>
+            <li>Mix sauces together</li>
+            <li>Heat pan/wok till slightly smoky</li>
+            <li>Add oil in then add Hor Fun noodles and add in 1 tsp of soy sauce. fry until brown on the edges</li>
+            <li>Remove the noodles</li>
+            <li>In the same pan add cooking oil add pork in.</li>
+            <li>Add chopped garlic then the prawns</li>
+            <li>Add in the chicken stock, bring to boil.</li>
+            <li>Add Bok Choy and stir in the corn starch solution.</li>
+            <li>Let it simmer for approximately 2 minutes</li>
+            <li>Add cooked hor fun and mix with the gravy and serve!</li>
+        </ol>
+    <p><img src="foodImages/hor-fun.jfif" alt="Delicious Hor Fun!" />" </p>
```

Comment:
If you need the img to behave like a block element, a div would be better suited here as paragraph usually contains text